base/Dockerfile

```diff
@@ -100,17 +100,20 @@ RUN apk add --no-cache \
     && apk del .build-dependencies \
     && rm -rf /usr/src/arp-scan
 
-# PicoTTS - it has no specific version - unfortunately we'll have to use master directly
-RUN apk add automake autoconf git libtool popt-dev git build-base \ 
-    && mkdir -p -m 755 /usr/src/pico \
-    && git clone https://github.com/naggety/picotts.git /usr/src/pico \
-    && cd /usr/src/pico/pico \
+# PicoTTS - it has no specific version - use current master branch commit
+ARG PICOTTS_HASH=e3ba46009ee868911fa0b53db672a55f9cc13b1c

```

Comment:
Push this into `build.json`

Reply:
Never tried to do it that way. Hope, it has been done correclty, could not test it.

I took my orientation from the arguments above. Hope it works.